Kickstarter-iOS/Views/Storyboards/LoadingBarButtonItemView.xib

```diff
@@ -20,7 +20,7 @@
                     <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
```

Comment:
Repositioned the activity indicator to aligned to the trailing edge of the `UIButton`, and right aligned the `titleLabel` within the `UIButton`